manual/cops_sorbet.md

```diff
@@ -509,6 +509,38 @@ sig { params(b: String, a: Integer).void }
 def foo(b:, a: 1); end
 ```
 
+## Sorbet/ObsoleteStrictMemoization
+
+Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+--- | --- | --- | --- | ---
+Enabled | Yes | No | - | -
```

Comment:
Why does `bundle exec rake generate_cops_documentation` think it doesn't support autocorrection? It implements `#autocorrect`.

Reply:
Ah it happened when I switched from `< RuboCop::Cop::Cop` to `< RuboCop::Cop::Base`, which also broke the (previously passing) tests